app/models/auction_ended.rb

```diff
@@ -0,0 +1,14 @@
+class AuctionEnded
```

Comment:
I like `AuctionEndedJob`. It's the generic home to any and all tasks that need to happen when the auction ends. Perhaps it could even call an `AuctionEndedEmailSender` object.